contract/src/lib.rs

```diff
@@ -4,19 +4,21 @@ use near_sdk::collections::UnorderedMap;
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, ext_contract, near_bindgen, AccountId, Balance, Promise, PromiseResult};
 use uint::construct_uint;
+use util::yton;
 //use std::collections::UnorderedMap;
 
 // a way to optimize memory management
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
+mod internal;
 mod nep21;
 mod util;
 
-// Prepaid gas costs. TODO: we need to adjust this value properly.
-const MAX_GAS: u64 = 200_000_000_000_000; // 100T gas
-const TX_NEP21_GAS: u64 = 20_000_000_000_000; // 20T gas
-
+// Prepaid gas for making a single simple call.
+const SINGLE_CALL_GAS: u64 = 200_000_000_000_000;
+const TEN_NEAR: u128 = 10_000_000_000_000_000_000_000_000;
+                               
```

Comment:
Please rollback this changes.

Reply:
Let me see the history
20Tgas is too low
Gas is refundable, no need to put low limits, at least not for the demo

Reply:
TEN_NEAR is used (refundable) when transferring NEP21. The fun tok contract might retain some near to back storage if the transfer generates a new acccount meaning more storage requirements for the contract. I'll adjust that down later.

Reply:
please rename it to nep21_storage_deposit (upper cased)